Dockerfile

```
@@ -0,0 +1,34 @@
+FROM ubuntu:latest
+
+RUN sysctl -w net.mptcp.enabled=1
+
+# Update package lists and install necessary dependencies
+RUN apt update && apt install -y \
+    python3 \
+    python3-pip \
+    python3.12-venv \
+    python3-apt \
+    openssl \
+    lighttpd \
+    iproute2 \
+    mptcpize \
+    autoconf automake libtool m4 pkg-config \
+    && rm -rf /var/lib/apt/lists/*
```

```suggestion
    && apt-get clean && rm -rf /var/lib/apt/lists/*
```

to remove `.deb` files as well